caddyhttp/proxy/proxy_test.go

```diff
@@ -247,8 +247,8 @@ func TestWebSocketReverseProxyNonHijackerPanic(t *testing.T) {
 func TestWebSocketReverseProxyServeHTTPHandler(t *testing.T) {
 	// No-op websocket backend simply allows the WS connection to be
 	// accepted then it will be immediately closed. Perfect for testing.
-	var connCount int32
```

Comment:
Part of the importance of this test, if I recall correctly, is to ensure that only one connection is made to the backend, hence the counter.

I think we shouldn't delete the count. I don't recall the issue anymore, and I wish the comments in the code mentioned it, but I do think the count is necessary.